src/main/java/com/auth0/json/mgmt/ResourceServer.java

```diff
@@ -7,19 +7,20 @@
 import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonProperty;
 
+@SuppressWarnings({"WeakerAccess", "unused"})
 @JsonIgnoreProperties(ignoreUnknown = true)
 @JsonInclude(JsonInclude.Include.NON_NULL)
 public class ResourceServer {
-    @JsonProperty
+    @JsonProperty("id")
     private String id;
-    @JsonProperty
+    @JsonProperty("name")
     private String name;
     @JsonProperty("identifier")
     private String identifier;
     @JsonProperty("scopes")
     private List<Scope> scopes;
-    @JsonProperty("signing_alg")
-    private String signingAlg;
+    @JsonProperty("signing_algorithm")
```

Comment:
the **json property** was correct `signing_alg`, what changes is the pojo field name (the line below)

Reply:
done